README.md

```diff
@@ -5,13 +5,21 @@
 
 ## Overview
 
-The [KubeVela](https://github.com/oam-dev/kubevela) User Experience (UX) Dashboard. Designed as an extensible, application-oriented delivery platform.
+VelaUX is a cloud-native application platform that leverages the KubeVela and Kubernetes ecosystem. It also functions as a plugin framework, enabling developers to design custom plugins and seamlessly integrate them into the KubeVela platform. This provides a flexible and customizable approach to expanding the platform's capabilities.
+
+### Highlights
+
+*Customizable User Interface*: With VelaUX, enterprises can customize the user interface for managing application and the infrastructure to meet their specific needs. This allows for a more intuitive and efficient user experience, leading to increased productivity and better utilization of resources.
```

Comment:
*Customizable User Interface*: With VelaUX, enterprises can tailor the user interface to their specific needs for managing applications and infrastructure. This feature leads to a more intuitive and efficient user experience, resulting in increased productivity and better resource utilization.